packages/fxa-auth-server/lib/payments/subscription-reminders.ts

```diff
@@ -158,7 +145,7 @@ export class SubscriptionReminders {
           acceptLanguage: account.locale,
         }
       );
-      await this.updateSentEmail(uid, emailParams);
+      this.updateSentEmail(uid, emailParams);
```

Comment:
Still needs the await here, otherwise the promise won't be waited on and context lost if it fails.